homeassistant/components/mutesync/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The mütesync integration."""
+from __future__ import annotations
+
+import asyncio
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+import mutesync
+from .const import DOMAIN
+
+# TODO List the platforms that you want to support.
+# For your initial PR, limit it to 1 platform.
+PLATFORMS = ["binary_sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the mutesync component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up mütesync from a config entry."""
+    # TODO Store an API object for your platforms to access
+    print(entry.data["token"])
+    print(entry.data["host"])
+    print("====================================")
+    hass.data[DOMAIN][entry.entry_id] = mutesync.PyMutesync(
```

Comment:
Don't use empty `async_setup`

Remove stale print

Remove stale comment

```suggestion
async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
    """Set up mütesync from a config entry."""
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = mutesync.PyMutesync(
```